client/src/theme/_uswds-theme-custom-styles.scss

```diff
@@ -290,6 +298,18 @@ textarea + .usa-radio > .usa-radio__label {
   }
 }
 
+.usa-input__wrapper {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  position: relative;
+  min-width: 5ex;
+  max-width: 14ex;
```

Comment:
Can we use `rem` instead of `ex` here?  I don't think we use that unit anywhere else.